### PR TITLE
Fix/4318 add more user-friendly name for publications

### PR DESCRIPTION
**Pull request by asf-algoo**
#4318 

## Checkpoints

**For developers**

- [x] If relevant, manual tests have been done to ensure the stability of the whole application and that the involved feature works
- [x] The original issue is up to date w.r.t the latest discussions and contains a short summary of the implemented solution
- [x] Automated tests covering the feature or the fix, have been written, deemed irrelevant (give the reason), or an issue has been created to implement the test (give the link)
Tests will be added at `fix/4254__add_tests_for_publication_page`
- [x] Make sure that:
  - if there are modifications in the Tracim configuration files (eg. `development.ini`), they are documented in `backend/doc/setting.md`
  - any migration process required for existing instances is documented
  - relevant people for these changes are notified
- [x] Original authors of the features included in a multi-feature branch (maintenance fixes -> develop, security fixes -> develop, …) should be part of the reviewers, especially if you encountered merge conflicts.

**For code reviewers**

- [x] The code is clear enough
- [x] If there are FIXMEs in the code, related issues are mentioned in the FIXME
- [x] If there are TODOs, NOTEs or HACKs in code, the date and the developer initials are present

**For testers**

- [x] Manual, quality tests have been done

### CODE REVIEW

File: frontend/i18next.scanner/en/translation.json [hunk 1]
```diff
@@ -425,5 +425,6 @@
   "Validate anyway": "Validate anyway",
   "Simple edition": "Simple edition",
   "Advanced edition": "Advanced edition",
-  "Publish": "Publish"
+  "Publish": "Publish",
+  "Publication of {{author}} at {{date}}": "Publication of {{author}} at {{date}}"
```
Comment (raphj) [inline frontend/i18next.scanner/en/translation.json:429]: I think "on" is more appropriate than "at" before an absolute date.

see for instance https://www.englisch-hilfen.de/en/words/date.htm (look for "complete date")

**Review by raphj — APPROVED** [review]

File: frontend/i18next.scanner/en/translation.json (hunk 1 above)
Comment (gbonaspetti) [inline frontend/i18next.scanner/en/translation.json:429]:
```suggestion
  "Publication of {{author}} on {{date}}": "Publication of {{author}} on {{date}}"
```

File: frontend/i18next.scanner/fr/translation.json [hunk 2]
```diff
@@ -425,5 +425,6 @@
   "Validate anyway": "Valider quand même",
   "Simple edition": "Édition simple",
   "Advanced edition": "Édition avancée",
-  "Publish": "Publier"
+  "Publish": "Publier",
+  "Publication of {{author}} at {{date}}": "Publication de {{author}} le {{date}}"
```
Comment (gbonaspetti) [inline frontend/i18next.scanner/fr/translation.json:429]:
```suggestion
  "Publication of {{author}} on {{date}}": "Publication de {{author}} le {{date}}"
```

File: frontend/i18next.scanner/pt/translation.json [hunk 3]
```diff
@@ -425,5 +425,6 @@
   "Validate anyway": "Validar de qualquer forma",
   "Simple edition": "Edição simples",
   "Advanced edition": "Edição avançada",
-  "Publish": "Publicar"
+  "Publish": "Publicar",
+  "Publication of {{author}} at {{date}}": "Publicação de {{author}} em {{date}}"
```
Comment (gbonaspetti) [inline frontend/i18next.scanner/pt/translation.json:429]:
```suggestion
  "Publication of {{author}} on {{date}}": "Publicação de {{author}} em {{date}}"
```

File: frontend/src/container/Publications.jsx [hunk 4]
```diff
@@ -293,11 +293,15 @@ export class Publications extends React.Component {
   handleClickValidateAnyway = async () => {
     const { props, state } = this
     const workspaceId = props.match.params.idws
-    const randomNumber = uuidv4()
+    const publicationName = props.t('Publication of {{author}} at {{date}}', {
```
Comment (gbonaspetti) [inline frontend/src/container/Publications.jsx:296]:
```suggestion
    const publicationName = props.t('Publication of {{author}} on {{date}}', {
```

**Review by Skylsmoi — APPROVED** [review]

**Pull request merged**